arm-network/2016-09-01/swagger/network.json

```diff
@@ -7900,6 +7898,43 @@
       },
       "description": "VpnClientParameters"
     },
+    "TunnelConnectionHealth": {
+      "properties": {
```

Comment:
Since all the properties of this object are  readOnly then can you please make that change in the swagger spec?